packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -138,7 +144,7 @@ class Plugin implements CLNPlugin {
     var result = HashMap<String, Object>();
     result["level"] = level;
     result["message"] = message;
-    stdout.write(jsonDecode(Response(result: result).toJson()));
+    stdout.write(Response(result: result).toJson());
```

Comment:
Sure that this works?

Reply:
The error on IDE disappears but when I try to log the `messageSocket` in start() method, this is returned `2022-06-16T16:00:01.315Z UNUSUAL plugin-cln_plugin_example.exe: Killing plugin: Malformed JSON-RPC notification missing "method" or "params": {"jsonrpc":"2.0","result":{"level":"debug","message":"Instance of 'Request'"}}
`

Reply:
maybe you need to put the also the id inside the constructor?

Somethings like `Response(id: 40, result: result).toJson()`